dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties

```diff
@@ -2043,8 +2043,10 @@ data_integrity.data_elements_aggregate_aggregation_operator.name=Non-numeric dat
 data_integrity.data_elements_aggregate_no_analysis.name=Aggregate data elements not used in any favourites (directly or through indicators)
 data_integrity.data_elements_aggregate_no_data.name=Aggregate data elements with NO data values.
 data_integrity.data_elements_aggregate_no_groups.name=Aggregate data elements not in any data element groups.
-data_integrity.data_elements_aggregate_with_different_period_types.name=Aggregate data elements which belong to datasets with different period types.
-data_integrity.datasets_empty.name=Datasets with no data elements.
+data_integrity.data_elements_aggregate_with_different_period_types.name=Aggregate data elements which belong to data sets with different period types.
```

Comment:
I'm also tempted to remove "." from all names. It does not make much sense to have punctuation in these names. 
But this can also be done through `tranifex`, I guess?